### PR TITLE
updating UI from source project

### PR DESCRIPTION
**Pull request by zeedevmobile**
once everything is updated i still need to go back ad polish a little, not able to test it currently

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -2,8 +2,8 @@ apply plugin: 'com.android.application'
 
 android {
 
-    compileSdkVersion rootProject.ext.compileSdkVersion
-    buildToolsVersion rootProject.ext.buildToolsVersion
+    compileSdkVersion 27
+    buildToolsVersion '27.0.3'
```
Comment (tir38) [inline app/build.gradle]: Gradle didn't like this?

File: app/src/main/java/io/orionlabs/android/interview/GroupListFragment.java [hunk 2]
```diff
@@ -26,16 +33,16 @@ public void onAttach(Context context) {
         ((OrionApplication) context.getApplicationContext()).getMainComponent().inject(this);
     }
 
-    @Nullable
     @Override
-    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
-        return inflater.inflate(R.layout.fragment_group_list, container, false);
+    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.view_drawer_group, container, false);
```
Comment (tir38) [inline app/src/main/java/io/orionlabs/android/interview/GroupListFragment.java]: wanna rename to  `view_group_list` ?

File: app/src/main/java/io/orionlabs/android/interview/GroupListFragment.java [hunk 3]
```diff
@@ -3,20 +3,27 @@
...
 import javax.inject.Inject;
 
-public class GroupListFragment extends Fragment implements GroupsListContract.PolicyDelegate {
+/**
+ * This will be the groups drawer and contain a list of groups with some fancy pants animations. For now it is just a placeholder.
+ */
+public class GroupListFragment extends Fragment implements GroupListContract.PolicyDelegate {
 
     @Inject
     GroupListPresenter presenter;
```
Comment (tir38) [inline app/src/main/java/io/orionlabs/android/interview/GroupListFragment.java]: Do you wanna do the same thing here and have this depend on the interface?

File: app/src/main/res/layout/list_item_group.xml [hunk 4]
```diff
@@ -53,27 +53,26 @@
...
 
-    <!--<com.onbeep.obiwan.component.SelectableImageView-->
-        <!--android:id="@+id/imageview_list_item_group_icon"-->
-        <!--android:layout_width="40dp"-->
-        <!--android:layout_height="40dp"-->
-        <!--app:layout_constraintBottom_toTopOf="@id/guideline_list_item_group_horizontal_begin"-->
-        <!--app:layout_constraintEnd_toStartOf="@+id/guideline_list_item_group_vertical_begin"-->
-        <!--app:layout_constraintStart_toStartOf="parent"-->
-        <!--app:layout_constraintTop_toTopOf="parent"/>-->
+    <com.onbeep.obiwan.component.SelectableImageView
```
Comment (tir38) [inline app/src/main/res/layout/list_item_group.xml]: did you pull this back into the project?

File: app/build.gradle (hunk 1 above)
Comment (zeedevmobile) [reply app/build.gradle]: the was probably from when i had to manually update the module settings for some reason

**Comment by zeedevmobile** [comment]
added a few updates to master:
1. Switched to latest MVP pattern
2. Cleaned up a couple minor UI issues
3. Added a couple more groups to make the list look better.

**Pull request merged**